modules/apps/headless/headless-delivery/headless-delivery-impl/src/main/java/com/liferay/headless/delivery/internal/dto/v1_0/converter/MessageBoardMessageDTOConverter.java

```diff
@@ -153,6 +161,30 @@ public MessageBoardMessage toDTO(DTOConverterContext dtoConverterContext)
 		};
 	}
 
+	private String _getFeaturedDomainName(long groupId, User user) {
+		try {
```

Comment:
You should remove the try/catch statement because the method outside it is already dealing with this exception

Besides that, everything is fine 
Great job 👍 😄